### PR TITLE
SSH Importer: Width fix to avoid cut of CheckBoxFrame Label

### PR DESCRIPTION
**Pull request by teclator**
By some reason the Label of the CheckBoxFrame is being cut. Now width is forced, but could be nice if some UI expert reviewed it as i'm not sure if could be some problem with long texts **(translations)**.

## Previously
ssh-key_dialog_new

## With MinWidth

ssh-key_dialog_new_length

### CODE REVIEW

File: src/lib/installation/dialogs/ssh_import.rb [hunk 1]
```diff
@@ -64,21 +64,23 @@ def copy_config
...
-              VSpacing(1),
-              HBox(
-                HSpacing(2),
-                Mode.config ? device_name : partitions_list_widget
-              ),
-              VSpacing(3),
-              HBox(
-                HSpacing(2),
-                Left(copy_config_widget)
+          MinWidth(50,
```
Comment (ancorgs) [inline src/lib/installation/dialogs/ssh_import.rb]: Obviously, the right fix is to discover why the cut is happening. But, once we are already in the business of workarounds... why don't make it i18n aware with something like...?

```ruby
label = _("I would like to import SSH keys from a previous installation")
MinWidth(label.length,
...
```

**Comment by ancorgs** [comment]
No version increase?

File: src/lib/installation/dialogs/ssh_import.rb [hunk 2]
```diff
@@ -62,23 +62,26 @@ def copy_config
...
-              VSpacing(1),
-              HBox(
-                HSpacing(2),
-                Mode.config ? device_name : partitions_list_widget
-              ),
-              VSpacing(3),
-              HBox(
-                HSpacing(2),
-                Left(copy_config_widget)
+          MinWidth(label.length,
```
Comment (ancorgs) [inline src/lib/installation/dialogs/ssh_import.rb]: Maybe add a comment explaining why this MinWidth is there

**Comment by jreidinger** [comment]
LGTM

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**